### PR TITLE
Add the ability to configure the postgres container

### PR DESCRIPTION
**Pull request by osaxma**
Hello again! 

I added few things that helps with testing in general that includes:
- mounting `pg_hba.conf`
- mounting `postgresql.conf`
- Also the user can pass individual configs using `configurations`

### CODE REVIEW

File: lib/containers/postgres.dart [hunk 1]
```diff
@@ -12,12 +24,45 @@ Future<DockerProcess> startPostgres({
   String? pgDatabase,
   int pgPort = 5432,
   bool? cleanup,
+  String? postgresqlConfPath,
+  String? pgHbaConfPath,
+  List<String>? configuraions,
```
Comment (isoos) [inline lib/containers/postgres.dart]: `configuraions` -> `configurations` (also in dartdoc)

**Review by isoos — CHANGES_REQUESTED** [review]
Small typo, otherwise looks good!
Please also add an entry in `CHANGELOG.md`.

File: lib/containers/postgres.dart (hunk 1 above)
Comment (osaxma) [reply lib/containers/postgres.dart]: Done -- I should have a spell check turned on next time 😅

**Review by isoos — APPROVED** [review]

**Comment by isoos** [comment]
published, thank you :)

**Pull request merged**